app/components/arcsi/Player.vue

```diff
@@ -169,6 +172,9 @@ export default {
     }
     this.setMetaData()
     this.$store.commit('player/currentlyPlayingArcsi', this.episode)
+    if (!(navigator.vendor.match(/apple/i) || '').length > 0) {
```

Comment:
@baxgas don't we want to pause if we have an Apple device (so `isSafari `instead of `not isSafari`)? Btw, can't you use iSafari here?

Reply:
@gammaw  I think on iOs all the browsers use Safari engine, just the "skin" is different, so the check should work for all. If you can try on a real device that would be the best

Reply:
Ok, I try to build the branch on the test server. 

But before doing so: 

You think that `vendor` would be Apple for the Chrome app on iPhone? 

But what I meant by my original question was that don't you want to pause the player on Apple devices? It seems to be reversed logic logic now.